src/components/loadJSON.js

```diff
@@ -5,19 +5,21 @@ var maxInFlight = 10;
 
 module.exports.create = function create_json_parse_stream(dataDirectory) {
   return parallelStream(maxInFlight, function(record, enc, next) {
-    fs.readFile(dataDirectory + record.path, function(err, data) {
+    var full_file_path = dataDirectory + record.path;
+    fs.readFile(full_file_path, function(err, data) {
       if (err) {
+        console.error('exception reading file ' + full_file_path);
         next(err);
       } else {
         try {
           var object = JSON.parse(data);
           next(null, object);
         } catch (parse_err) {
-          console.error('exception on %s:', record.path, parse_err);
+          console.error('exception parsing JSON in file %s:', record.path, parse_err);
           console.error('Inability to parse JSON usually means that WOF has been cloned ' +
                         'without using git-lfs, please see instructions here: ' +
                         'https://github.com/whosonfirst/whosonfirst-data#git-and-large-files');
-          next(null, {});
+          next(parse_err);
         }
```

Comment:
Lint is complaining about the long lines, that's why I changed them.  I'll disable that in .jslintrc for now.